changelogs/fragments/2008-update-java-cert-replace-cert-when-changed.yml

```
@@ -0,0 +1,7 @@
+minor_changes:
+  - "java_cert: Change ``state: present`` to check certificates by hash, not just alias name (https://github.com/ansible/ansible/issues/43249)."
+bugfixes:
+  - "java_cert: Allow setting ``state: absent`` by providing just the cert_alias - (https://github.com/ansible/ansible/issues/27982)."
+  - "java_cert: Properly handle proxy arguments when the scheme is provided - (https://github.com/ansible/ansible/issues/54481)."
```

```suggestion
  - "java_cert - properly handle proxy arguments when the scheme is provided (https://github.com/ansible/ansible/issues/54481)."
```
Same for the above lines.